integration-tests/tests/janitor.js

```diff
@@ -0,0 +1,44 @@
+const rawConsole = require('console')
+const dateFns = require('date-fns/fp')
+const _ = require('lodash/fp')
+const { testWorkspaceNamePrefix } = require('../utils/integration-helpers')
+const { dismissNotifications, signIntoTerra } = require('../utils/integration-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+const olderThanDays = 2
+
+const runJanitor = withUserToken(async ({ billingProject, page, testUrl, token }) => {
+  // Sign into Terra so we have the correct credentials.
+  await page.goto(testUrl)
+  await signIntoTerra(page, token)
+  await dismissNotifications(page)
+
+  // Delete old orphaned workspaces. These can result from local integration test runs where the
+  // process was prematurely terminated, but a few also appear in alpha and staging indicating that
+  // ci test runs also sometimes leak workspaces.
```

Comment:
It might be good to move this comment higher, after the `require`s, and perhaps to also include a sentence about why clean-up was implemented as a test.

Reply:
I can imagine the test being used to clean up other things in the future, so I think it's best to keep the comment near the related code. And the history of all of this is accessible via git (ticket, PR, etc.).

Reply:
It helps to be explicit about reasonings, especially in codebases contributed to by so many people but I won't push further 😊